src/assets/styles/themes/base.scss

```diff
@@ -78,8 +78,7 @@
 
   // TODO: discuss with UX
   /* colors rgb */
-  --rp-ui-base-almost-black-light: rgba(63, 63, 63, 0.75);
-  --rp-ui-base-black: rgba(34, 34, 34, 0.91);
+  --rp-ui-tooltip-bg: rgba(34, 34, 34, 0.91);
```

Comment:
All colors in this file should have prefix `base`.

Reply:
Done.